Dockerfile

```diff
@@ -112,5 +123,7 @@ ENV PATH /usr/src/app/bin:$PATH
 EXPOSE 5000
 VOLUME /workspace
 
+RUN gcloud auth configure-docker
```

Comment:
It can be run from within here as well as the freight user. Is this an operation that only needs to happen once? If so, it'd be worth doing it inside the Dockerfile, as long as we just execute it as the freight user imo.

Reply:
My thought process was that it's of little-to-negative value to run it for anyone who isn't using GCR. The things it does are relatively idempotent... I itemized them in the Jira ticket.

Reply:
Ah, ok. Is this process slow? I thought it needed to download and install something? If so, the concern of doing it in entrypoint is that it'll now make running each container X seconds slower, while it downloads something. And also open for failure at that point. Assuming it does what I think it does?

Reply:
No.  It checks to see that gcloud and docker are installed (literally `[ -f ... ]`) and then appends some json to the user's docker config file.

Reply:
Ah, cool, yeah, then slam it also in the entrypoint.